docs/architecture/_index.md

```diff
@@ -0,0 +1,192 @@
+
+When you control your machine or view its camera streams or sensor outputs from the Viam app **CONTROL** tab, those connections happen over WebRTC.
+The Viam app hits the same API endpoints as your SDK client code, with `viam-server` handling requests.
+
+{{% alert title="Protobuf APIs" color="info" %}}
+All Viam APIs are defined with the [Protocol Buffers (protobuf)](https://protobuf.dev/) framework.
+This is what enables Viam to provide SDKs in a variety of different programming languages.
+{{% /alert %}}
+
+For more details, see [Machine-to-Machine Communication](/architecture/machine-to-machine-comms/).
+
+### Security
+
+TLS certificates certificates provided by the Viam app ensure that all communication is authenticated and encrypted.
+
+Viam uses API keys with [role-based access control (RBAC)](/fleet/rbac/) to control access to machines from client code.
+
+## Data management flow
+
+{{<imgproc src="/architecture/data-flow.svg" resize="x1100" declaredimensions=true alt="Data flowing from local disk to cloud to the Viam app, SDKs, and MQL and SQL queries." >}}
+<br>
+
+Here's how data flows in Viam:
+
+1. Data collected by your components, such as sensors and cameras, is first stored locally in a specified directory (defaults to <file>~/.viam/capture</file>).
+   You control how often to capture data and where to store it using the configuration file.
+
+   - You can also sync data from other sources by putting it into this folder.
+     <br><br>
+
+1. `viam-server` syncs data to a MongoDB database in the cloud at your specified interval, and deletes the data from the local directory.
+
+1. You can view your data from the Viam app or query it using Viam SDKs, MQL, or SQL.
+
+If a device has intermittent internet connectivity, no data is lost because it is stored locally until the machine can reconnect to the cloud.
+
+For more information, see [Data Management](/services/data/).
+
+## Basic machine example
+
+<p>
+{{<imgproc src="/architecture/simple-machine.png" class="alignright" resize="x1100" declaredimensions=true alt="viam-server running on a board connected to a sensor. Data is stored on a local folder and synced to a folder in the Viam app cloud." style="max-width:400px" >}}
+</p>
+
+Imagine you have a simple device consisting of a temperature sensor connected to the GPIO pins of a single-board computer (SBC).
+You want to capture sensor data at regular intervals, and sync it to the cloud.
+Here is how this works in Viam:
+
+- You create a configuration file in the Viam app that includes:
+  - A sensor {{< glossary_tooltip term_id="component" text="component" >}}
+    - Configure which pins of the SBC the sensor is connected to.
+  - Data management {{< glossary_tooltip term_id="service" text="service" >}}
+    - Configure the intervals at which to capture and sync data.
```

Comment:
remove unnecessary detail
```suggestion
- You create a configuration file in the Viam app that includes a sensor {{< glossary_tooltip term_id="component" text="component" >}} and the Data management {{< glossary_tooltip term_id="service" text="service" >}}.
```

Reply:
I included this to give people an idea of what the configuration even is